packages/formstr-app/src/containers/CreateFormNew/components/QuestionCard/index.tsx

```
@@ -88,8 +88,8 @@ const QuestionCard: React.FC<QuestionCardProps> = ({
                 key={question[1]}
                 className="question-input"
                 onChange={handleTextChange}
-                value={question[3] || "Click to edit"}
-                placeholder="Enter a Question"
+                value={question[3]}
+                placeholder="Click here to edit"
```

this should be solved via defaultValue prop instead of a placeholder, since this is misleading, you can incorporate the changes here:  https://github.com/keraliss/nostr-forms/pull/1/files#diff-3d18616507c1d65b891ed65ed0f879f4e84b7c9dffad0530486de7bd6d4ce617

---

Okay will do that